sourcehut-dark.css

```diff
@@ -48,44 +51,6 @@ a {
     color: black;
 }
 
-.btn.btn-primary.btn-block {
-    color: var(--soft-weiss);
-}
-
-body {
-    background-image: var(--ghd-bg-img);
```

Comment:
I tested and the css was missing the background. This line is removed but not re-added. I put another block with this and it worked. If you can do that, I can merge.